.github/workflows/debian-trust-package-release-bookworm.yaml

```diff
@@ -16,6 +16,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+      # Adding `fetch-depth: 0` makes sure tags are also fetched. We need
+      # the tags so `git describe` returns a valid version.
+      # see https://github.com/actions/checkout/issues/701 for extra info about this option
+      with: { fetch-depth: 0 }
```

Comment:
```suggestion
      with:
        fetch-tags: true
```

Reply:
I was blindly copying the other job but this definitely seems worth testing, thanks :D

I guess the comment is out of date, I'll update that and repush